js/piwik.js

```diff
@@ -5148,9 +5219,9 @@ if (typeof Piwik !== 'object') {
                  * @param string name (optional) The Event's object Name (a particular Movie name, or Song name, or File name...)
                  * @param float value (optional) The Event's value
                  */
-                trackEvent: function (category, action, name, value) {
+                trackEvent: function (category, action, name, value, callbacks) {
```

Comment:
Yup we can use ` onSuccess, onError` 1 more argument to the function. I doesn't break the compatibility cf L2513

```js
 // Compatibility for eventLink
       if('function' === typeof cb) {
```